app/src/main/kotlin/com/mapzen/erasermap/model/MapzenLocationImpl.kt

```diff
@@ -38,11 +36,11 @@ public class MapzenLocationImpl(val locationClient: LostApiClient,
     private fun connect() {
         if (!locationClient.isConnected) {
             locationClient.connect()
-
-            if (settings.isMockLocationEnabled) {
-                LocationServices.FusedLocationApi?.setMockMode(true)
-                LocationServices.FusedLocationApi?.setMockLocation(settings.mockLocation)
-            }
+        }
+        if (settings.isMockLocationEnabled) {
+            LocationServices.FusedLocationApi?.setMockMode(true)
+            LocationServices.FusedLocationApi?.setMockLocation(settings.mockLocation)
+            LocationServices.FusedLocationApi?.setMockTrace(settings.mockRoute)
```

Comment:
Nice catch. I've pushed changes to check this setting. I also updated the way our settings behave so that enabling mock routing enables mock location as well.